homeassistant/components/tuya/__init__.py

```diff
@@ -153,6 +153,17 @@ async def async_setup_entry(hass: HomeAssistant, entry: TuyaConfigEntry) -> bool
     # Register known device IDs
     device_registry = dr.async_get(hass)
     for device in manager.device_map.values():
+        if not device.status and not device.status_range and not device.function:
+            # If the device has no status, status_range or function,
+            # it cannot be supported
+            LOGGER.info(
```

Comment:
This should not have been logger at INFO level. A user can’t do anything about this situation.

Reply:
You can't make that call with the information in here. A user would need to contact Tuya support, as they often can standardize the device.

Additionally, I don't get why we would log this, but still create a device?
This feels all duplicate.

../Frenck

Reply:
> I'll try to link if something... but I think it's worth trying to differentiate between something that can be actioned by HA maintainers and something that needs official Tuya support

We can't know; could be both.

../Frenck